gnd/src/main/java/com/google/android/gnd/repository/ProjectRepository.java

```diff
@@ -97,6 +99,12 @@ public ProjectRepository(
         .compose(Loadable::loadingOnceAndWrap);
   }
 
+  /**
+   * Loads the project with the given id from the remote data store and saves it to the local data
+   * store
+   * @param id - id of project to load
+   * @return A Single representing the project (@see http://reactivex.io/documentation/single.html)
```

Comment:
If we consider Single as low-level types like `Optional` and `Consumer`, would it make sense to count on the user referring to the JavaDoc instead of pointing them to additional docs? We use Rx very widely, and it may become burdensome to consistently link to Rx docs.

Reply:
Good point. I will remove the link. A base level of RxJava knowledge is required to work with the codebase and it doesn't make sense for us to teach that to the user.